hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneMultipartUploadList.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.ozone.client;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * List of in-flight MU upoads.
+ */
+public class OzoneMultipartUploadList {
+
+  private List<OzoneMultipartUpload> uploads = new ArrayList<>();
```

Comment:
Unneeded initialization.
As, we are setting uploads with value passed to constructor.

Do we need to have Not null check here below?

```
  public OzoneMultipartUploadList(
      List<OzoneMultipartUpload> uploads) {
    this.uploads = uploads;
```